frappe/email/doctype/standard_reply/standard_reply.json

```diff
@@ -13,6 +14,37 @@
  "engine": "InnoDB", 
  "fields": [
   {
+   "allow_bulk_edit": 0, 
+   "allow_on_submit": 0, 
+   "bold": 0, 
+   "collapsible": 0, 
+   "columns": 0, 
+   "fieldname": "name", 
```

Comment:
please remove this!!!!

Reply:
@rmehta , removed and used "Prompt" for naming instead. Thanks!